sample-customisation/src/main/res/values-es/beacon-override.xml

```diff
@@ -0,0 +1,10 @@
+<resources>
+  <!--
+    This file demonstrates overriding some of the Beacon strings for Español
+    See https://developer.helpscout.com/beacon-2/android/#translation-strings for full list
+   -->
+  <string name="hs_beacon_answers">Respuestas</string>
+  <string name="hs_beacon_search_hint">Pista de búsqueda</string>
+  <string name="hs_beacon_nothing_found">Nada Encontrado</string>
```

Comment:
Currently. in the app we have these values set to:
"What can we help you with?"
"We couldn't find any articles that match your search."

So I've translated those to Spanish.

```suggestion
  <string name="hs_beacon_search_hint">¿Cómo te podemos ayudar?</string>
  <string name="hs_beacon_nothing_found">No hemos encontrado ningún artículo que coincida con tu búsqueda</string>
```